scripts/datascience/themes_quality/config/pipeline_config.yaml

```
@@ -0,0 +1,148 @@
+  creature:
+    description: "Living creatures (real or fictional), species, or common names"
+    prototype_words: ["dog", "bird", "fish", "insect", "mammal", "cat", "horse", "cow", "pig", "sheep", "chicken", "duck", "eagle", "lion", "tiger", "elephant", "bear", "wolf", "deer", "rabbit", "mouse", "rat", "snake", "lizard", "frog", "whale", "dolphin", "shark", "turtle", "octopus", "crab", "lobster", "salmon", "trout", "butterfly", "bee", "bat", "owl", "hawk", "parrot", "penguin", "ostrich", "fox", "raccoon", "squirrel", "chipmunk", "beaver", "otter", "seal", "monkey", "gorilla", "chimpanzee", "kangaroo", "koala", "crocodile", "alligator", "iguana", "gecko", "salamander", "toad", "spider", "ant", "fly", "mosquito", "cricket", "grasshopper", "goat", "donkey", "llama"]
+    confidence_threshold: 0.3
+    dictionary_weight: 0.4
+    
+  skill:
+    description: "Capabilities, skills, or expertise"
+    prototype_words: ["programming", "writing", "speaking", "leadership", "creativity", "communication", "management", "teaching", "cooking", "driving", "singing", "dancing", "painting", "drawing", "reading", "listening", "thinking", "problem", "solving", "analysis", "research", "planning", "organizing", "negotiating", "presenting"]
+    confidence_threshold: 0.4
+    dictionary_weight: 0.5
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)